priv/static/app.js

```diff
@@ -0,0 +1,668 @@
+(function(/*! Brunch !*/) {
```

Comment:
This file should likely be priv/static/phoenix.js. We need to change the brunch-config in Phoenix itself.